docs-ref-conceptual/authenticate-azure-cli.md

```diff
@@ -15,14 +15,12 @@ keywords: az login, authentication types , authentication methods, azure, cli lo
 # Sign in with Azure CLI
 
 There are several authentication types for the Azure Command-Line Interface (CLI), so how do you log in?  The easiest way to get started is with [Azure Cloud Shell](/azure/cloud-shell/overview), which automatically logs you in.
-Locally, you can sign in interactively through your browser with the [az login](/cli/azure/reference-index#az-login) command. When writing scripts, the recommended approach is
+Locally, you can sign in interactively through your browser with the [az login](/cli/azure/reference-index#az-login) command. When you write scripts, the recommended approach is
 to use service principals. By granting just the appropriate permissions needed to a service principal, you can keep your automation secure.
 
-None of your login information is stored by Azure CLI. Instead, an [authentication refresh token](/azure/active-directory/develop/v1-id-and-access-tokens#refresh-tokens)
-is generated by Azure and stored. As of August 2018 this token is revoked after 90 days of inactivity, but this value can be changed by Microsoft or your tenant administrator. Once the token is revoked
-you get a message from the CLI saying you need to login again.
+Azure CLI doesn't store any of your login information. Instead, Azure CLI generates and stores a [authentication refresh token](/azure/active-directory/develop/v1-id-and-access-tokens#refresh-tokens). As of August 2018 this token is revoked after 90 days of inactivity. Contact your Microsoft or tenant administrator to change this value. Once the token is revoked, you get a message from the CLI saying you need to log in again.
```

Comment:
@daphnemamsft , Acrolinx is confused.  `an authentication` is correct.  Acrolinx is not overlooking the `[`.

Reply:
This has been revised